src/managers/tls.py

```diff
@@ -41,14 +41,30 @@ def import_ca(self, certificate: str):
         """
         self.workload.write(certificate, self.workload.paths.cert)
 
-        command = f"{self.workload.paths.keytool} -import -v -alias ca -file {self.workload.paths.cert} -keystore {self.workload.paths.truststore} -storepass {self.truststore_password} -noprompt"
+        command = [
+            self.workload.paths.keytool,
+            "-import",
+            "-v",
+            "-aliasca-file",
```

Comment:
you need to split this line :
```suggestion
            "-alias", 
            "ca", 
            "-file",
```

Reply:
yes, indeed. Let's give it a go! Great catch and thank you for noticing this!!!